Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -426,11 +426,13 @@ def sum_item(value, border=""):
     def send_report(self):
         if self.results is not None and self.selected_learner:
             index = self.selected_learner[0]
+            cmatrix = confusion_matrix(self.results, index)
+            n= len(cmatrix)
```

Comment:
I'm not certain about this, but if you have 1e9 examples that belong to three distinct classes, you have to make at least 1e9 operations to compute the confusion matrix ...

Or what did you mean?